### PR TITLE
Upgrade to n-ui v4

### PR DESCRIPTION
**Pull request by keirog**

### CODE REVIEW

File: client/main.scss [hunk 1]
```diff
@@ -1,23 +1,13 @@
...
-@import "n-ui/bootstrap";
-
-@include nUiCriticalStart();
-	@import 'n-image/main';
-@include nUiCriticalEnd();
-
+$o-buttons-is-silent: false !default;
+@import 'o-buttons/main';
+
+@import "n-ui/main";
```
Comment (debugwand) [inline client/main.scss]: I might have got the wrong end of the stick but I thought that `@import 'n-ui-foundations/main';` replaced n-ui/main? or is it different?

Comment (keirog) [inline client/main.scss]: Oh I thought that was just for components

File: bower.json [hunk 2]
```diff
@@ -1,8 +1,9 @@
 {
   "name": "ft-next-tour-page",
   "dependencies": {
-    "n-ui": "^3.0.0",
+    "n-ui": "^4.0.0-beta.2",
     "n-image": "^4.2.3",
-    "n-section": "^6.3.7"
+    "n-section": "^7.0.0",
+    "o-buttons": "^4.5.4"
```
Comment (wheresrhys) [inline bower.json]: not needed as it's a dependency of n-ui-foundations.

No biggie, and hopefully will add tooling to remove the guesswork from this

File: client/main.scss [hunk 3]
```diff
@@ -1,23 +1,13 @@
...
-);
-
-@import "n-ui/bootstrap";
-
-@include nUiCriticalStart();
-	@import 'n-image/main';
-@include nUiCriticalEnd();
-
+$o-buttons-is-silent: false !default;
+@import 'o-buttons/main';
```
Comment (wheresrhys) [inline client/main.scss]: Is this needed in critical css? Try deleting the <link rel=stylesheet> from the <head> of the page to see if anything in first view breaks

Comment (keirog) [inline client/main.scss]: Yep there’s a button at the top of the page

**Pull request merged**